pythonforandroid/recipes/attrs/__init__.py

```diff
@@ -0,0 +1,11 @@
+from pythonforandroid.recipe import PythonRecipe
```

Comment:
Do these python modules actually need recipes? I would have thought that the pip install would be fine in most cases? Is it because they depend on setuptools?

Reply:
Honestly I have no idea. I just thought adding recipes is the way to go to get packages into the build. But I fear that packages requiring ``call_hostpython_via_targetpython = False`` due to the use of ``io.so`` (IIRC) need dedicated recipes. And if it's true that recipes need to be done for packages depending on setuptools, they'd need to stay anyway.

Reply:
I'd prefer not to add recipes that aren't actually necessary, you can just try removing the recipes and seeing if the modules still work. In the medium term we can also look to improve the way the python build is done, the defaults are an unnecessary legacy of the old toolchain in some places.

Reply:
removed recipes, added to requirements of buildozer.spec. still works. BTW can i define package version via buildozer.spec requirements?

Reply:
Yep, you can use the pip syntax `package==version.number` - everything that doesn't have a recipe is actually passed straight to pip, so there are other things you can do as well.